core/src/epicli/data/common/ansible/playbooks/roles/kubernetes_common/tasks/install-packages.yml

```diff
@@ -1,12 +1,38 @@
 ---
 - name: Install Storage packages
   package:
-    name: "{{ item }}"
-    update_cache: true
+    name: "{{ _packages[ansible_os_family] }}"
     state: present
-  loop:
-    - cifs-utils
-    - samba-client
-    - samba-common
+  vars:
+    _packages:
+      Debian:
+        - cifs-utils
+        - samba-client
+        - samba-common
+        - nfs-common
+      RedHat:
+        - cifs-utils
+        - samba-client
+        - samba-common
+        - nfs-utils
```

Comment:
Haven't seen it yet, but what do you think about how it should look like when there is a long common list of packages?

Reply:
If it's really long we could just do set arithmetic (union) in jinja2, but here I think it's not worth it :thinking:

Reply:
Agree.